conf/main/mindmaps-engine.properties

```diff
@@ -48,4 +48,4 @@ backgroundTasks.time-lapse=60000
 loader.threads=0
 
 #MindmapsDB version
-project.version=0.1.0
+project.version=0.2.0-SNAPSHOT
```

Comment:
Can't this information be retrieved from elsewhere ? i.e. we should be able to delete this line ? @aelred correct me I am wrong but you mentioned this no ?